tests/tests_pytorch/callbacks/progress/test_rich_progress_bar.py

```diff
@@ -114,7 +114,7 @@ def test_rich_progress_bar_custom_theme(tmpdir):
         theme = RichProgressBarTheme()
 
         progress_bar = RichProgressBar(theme=theme)
-        progress_bar.on_train_start(Trainer(tmpdir), BoringModel())
+        progress_bar.on_train_start(Trainer(default_root_dir=tmpdir), BoringModel())
```

Comment:
we were passing `Trainer(logger=tmpdir)` 🤣

Reply:
It clearly did not run then. I suggest removing the tmpdir fixture instead for that 5ms time save